packages/kbn-shared-gen-ai-ui/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/shared-gen-ai-ui
```

Comment:
If possible I would put this package in `x-pack/packages` instead of the root packages since its only going to be used by `x-pack/plugins` is I understand correctly